projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn.json

```
@@ -0,0 +1,646 @@
+    "gui.refurbished_furniture.package_message": "包裹消息",
+    "gui.refurbished_furniture.send": "发送",
+    "gui.refurbished_furniture.how_to": "如何操作",
+    "gui.refurbished_furniture.post_box_info": "从列表中选择一个邮箱。你可以通过名称搜索邮箱，或者通过在名称前加上@来搜索玩家。将物品放入包裹槽中，并可选择包含一条消息。点击发送按钮以递送包裹。",
+    "gui.refurbished_furniture.workbench_info": "从列表中选择一个合成配方，并收集所需的材料来制作物品。工作台会搜索你的背包、自身存储以及相邻的存储方块中的材料。",
+    "gui.refurbished_furniture.package_sent_by": "由%s发送",
+    "gui.refurbished_furniture.package_open": "右键打开",
+    "gui.refurbished_furniture.set_doorbell_name": "设置门铃名称",
+    "gui.refurbished_furniture.doorbell_rang": "门铃响了",
+    "gui.refurbished_furniture.status.online": "在线",
+    "gui.refurbished_furniture.status.offline": "离线",
+    "gui.refurbished_furniture.status.overloaded": "过载",
+    "gui.refurbished_furniture.status.no_fuel": "燃料耗尽",
+    "gui.refurbished_furniture.node_count": "%s / %s",
+    "gui.refurbished_furniture.recycle": "回收",
+    "gui.refurbished_furniture.save": "保存",
+    "gui.refurbished_furniture.next_preset": "下一个预设",
+    "gui.refurbished_furniture.previous_preset": "上一个预设",
+    "gui.refurbished_furniture.no_power": "缺少电力",
+    "gui.refurbished_furniture.link_too_long": "距离过远",
+    "gui.refurbished_furniture.link_too_many": "链接过多",
+    "gui.refurbished_furniture.link_already_connected": "已连接",
+    "gui.refurbished_furniture.link_invalid_node": "无效节点",
+    "gui.refurbished_furniture.link_unpowerable": "链路无法供电",
+    "gui.refurbished_furniture.link_outside_area": "链路超出供电区域",
+    "gui.refurbished_furniture.connect_to_power": "确保方块从%s接收电力，可以使用%s进行连接。",
+    "gui.refurbished_furniture.electricity_generator": "发电机",
+    "gui.refurbished_furniture.jei_campfire_info": "继承自篝火烹饪",
+    "gui.refurbished_furniture.progress": "%s %s %s",
+    "gui.refurbished_furniture.hold_for_details": "按住%s查看详情",
+    "gui.refurbished_furniture.shift": "SHIFT",
+    "gui.refurbished_furniture.show_all_categories": "显示全部",
+    "gui.refurbished_furniture.mail_box_limit": "邮箱数量已达上限",
+    "gui.refurbished_furniture.booting": "启动中...",
+    "gui.refurbished_furniture.sliceable": "可切片",
+    "gui.refurbished_furniture.placeable": "可放置",
+    "gui.refurbished_furniture.workbench.search_neighbours.off": "忽略相邻容器",
+    "gui.refurbished_furniture.workbench.search_neighbours.on": "包含相邻容器",
+    "gui.refurbished_furniture.experience_level": "%s/%s 级",
+    "gui.refurbished_furniture.experience_points": "%s 点",
+    "gui.refurbished_furniture.withdraw_experience": "提取经验",
+    "gui.refurbished_furniture.requires_power": "需要从%s供电",
+    "subtitle.refurbished_furniture.package_open": "纸箱：撕裂",
+    "subtitle.refurbished_furniture.chair_slide": "木头：滑动",
+    "subtitle.refurbished_furniture.doorbell_chime": "门铃：鸣响",
+    "subtitle.refurbished_furniture.electricity_generator_engine": "发电机",
+    "subtitle.refurbished_furniture.storage_jar_insert_item": "物品：移动",
+    "subtitle.refurbished_furniture.recycle_bin_engine": "碾压声",

```

```suggestion
    "subtitle.refurbished_furniture.recycle_bin_engine": "碾压",
```
这些个人建议不需要“声”字，原版的`啜饮`等也没有